index.ts

```diff
@@ -36,11 +36,11 @@ function parseProjectName(repoName: string): string {
         .replace("/", "$");
 }
 
-function runElmApp(viewHash: string, dirPath: string, model: any): Promise<string> {
+function runElmApp(moduleHash: string, viewHash: string, dirPath: string, model: any): Promise<string> {

```

Comment:
🎨 what's the purpose of the viewHash if we're using a module hash instead? Shouldn't it be a combination of moduleName + viewName?